forge-game/src/main/java/forge/game/staticability/StaticAbilityAlternativeCost.java

```diff
@@ -36,14 +37,19 @@ public static List<SpellAbility> alternativeCosts(final SpellAbility sa, final C
                 newSA.setActivatingPlayer(pl);
                 newSA.setBasicSpell(false);
 
+                if (cost.hasXInAnyCostPart()) {
+                    newSA.setSVar("X", stAb.getSVar("X"));
+                }
+
+                // makes new SpellDescription
+                final StringBuilder sb = new StringBuilder();
+
                 // CostDesc only for ManaCost?
                 if (sa.isAbility()) {
                     newSA.putParam("CostDesc", stAb.hasParam("CostDesc") ? ManaCostParser.parse(stAb.getParam("CostDesc")) : cost.toSimpleString());
+                    sb.append(newSA.getCostDescription());
                 }
 
-                // makes new SpellDescription
-                final StringBuilder sb = new StringBuilder();
-                sb.append(newSA.getCostDescription());
                 // skip reminder text for now, Keywords might be too complicated
                 //sb.append("(").append(newKi.getReminderText()).append(")");
                 if (sa.isSpell()) {
```

Comment:
i think you can remove the `sb.append(" ")` below too

i had it before i added the cost into the Reminder Text